plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/writer/S3NativeTransactionLogSynchronizer.java

```diff
@@ -46,7 +46,7 @@
 import static java.util.Objects.requireNonNull;
 
 /**
- * The S3 Native synchronizer is a {@link TransactionLogSynchronizer} for S3 that requires no other dependencies.
+ * The S3 native synchronizer is a {@link TransactionLogSynchronizer} for S3 that requires no other dependencies.
```

Comment:
`S3ConditionalWriteLogSynchronizer` does not require other dependencies but the storage as well.

Databricks historically used a mechanism based on an online database to synchronize the access for writing to S3 for the Databricks clusters. This is where probably the following comment originates:

> that requires no other dependencies

I think that the comment is not relevant anymore. Also the current name of the class is not relevant anymore.

Suggestion:
`S3NativeTransactionLogSynchronizer` -> `S3LockTransactionLogSynchronizer`

Reply:
Yes, I am aware. I was there, 10000 years ago 😀